code/modules/hydroponics/spreading/spreading_response.dm

```diff
@@ -20,6 +20,8 @@
 /obj/effect/plant/Crossed(var/mob/living/victim)
```

Comment:
I wonder why this is typecasts towards living when only human mobs have a right/left leg/foot.